client/src/app/pages/importer-list/importer-list.tsx

```diff
@@ -250,14 +251,21 @@ export const ImporterList: React.FC = () => {
                           modifier="truncate"
                           {...getTdProps({ columnKey: "start" })}
                         >
-                          {formatDate(item.report?.startDate)}
+                          {item.state &&
+                            configValues?.disabled == false &&
+                            dayjs(item.report?.startDate)
+                              .utc()
+                              .format("YYYY-MM-DD HH:mm:ss")}
```

Comment:
Just an idea:
-  The current code in the main branch has a function `formatDate()` that can be reused everywhere.
- what about having something like `formatDateTime` so we can reuse it too?

Reply:
@carlosthe19916, 

Well I suppose this a question of reuse vs performance.
I'm not against a format function here but what's do we gain as the call to dayjs format is straightforward here.

Reply:
Having formats in a single place will help us to change the formats in a single shot when needed. So far this is the only place where we use this format but in the case of `formatDate` if UX decides the format needs to change We will need to only change that function and the whole app will have the desired format rather than changing `.format(ABCAD)` in every single file